### PR TITLE
message for the github workflow has been rewritten properly

### PR DESCRIPTION
**Pull request by saurabh100ni**
**Description**

This PR fixes the [message for GitHub workflows](https://layer5.io/cloud-native-management/gitops/snapshot#:~:text=See%20this%20sample%20GitHub%20workflow%20(action.yml)%20for%20more%20configuration%20details.) by writing it properly.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 62fa99939dcd3e5b95558aea0db29bd4ff899893 at: https://65453b70a152900f36cc6ad8--layer5.netlify.app

File: src/sections/gitops/PerformanceManagementPage.js [hunk 1]
```diff
@@ -87,7 +87,7 @@ const PerformanceManagementPage = () => {
       <ContentContainerWrapper>
         <ContentRow>
           <TextColumn>
-            <Heading>See this sample GitHub workflow (action.yml) for more configuration details.</Heading>
+            <Heading>See this sample GitHub workflow in the 'action.yml' for configuration details.</Heading>
```
Comment (leecalcote) [inline src/sections/gitops/PerformanceManagementPage.js:90]: Thanks, but where is the action.yml link?

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: src/sections/gitops/PerformanceManagementPage.js (hunk 1 above)
Comment (leecalcote) [reply src/sections/gitops/PerformanceManagementPage.js:90]: If not a link then, a local example.

If neither of these, then a different message in this section is needed.

Comment (saurabh100ni) [reply src/sections/gitops/PerformanceManagementPage.js:90]: I was confused about which action.yaml file you were referring to, although I found this [action.yaml](https://github.com/layer5labs/meshmap-snapshot/blob/master/action.yml) file, please confirm if it is the one you are referring to.

**Comment by saurabh100ni** [comment]
I am closing this PR and Opening a new Issue for this as a New Design for that Section is needed.

**Pull request closed without merging**